lzero/mcts/buffer/game_segment.py

```diff
@@ -128,7 +131,8 @@ def append(
             obs: np.ndarray,
             reward: np.ndarray,
             action_mask: np.ndarray = None,
-            to_play: int = -1
+            to_play: int = -1,
+            chance: np.ndarray = 0,
```

Comment:
why is the default value a python scalar

Reply:
fixed now